pennylane/ops/op_math/controlled_class.py

```diff
@@ -29,6 +29,27 @@
 
 from .symbolicop import SymbolicOp
 
+# pylint: disable=protected-access
+def _decompose_no_control_values(op: "operation.Operator") -> List["operation.Operator"]:
+    """Provides a decomposition without considering control values.  Returns None if
+    no decomposition.
+    """
+    if len(op.control_wires) == 1 and hasattr(op.base, "_controlled"):
+        return [op.base._controlled(op.control_wires[0])]
+    if isinstance(op.base, qml.PauliX):
+        if len(op.control_wires) == 2:
+            return [qml.Toffoli(op.active_wires)]
+        return [qml.MultiControlledX(wires=op.active_wires, work_wires=op.work_wires)]
+
+    try:
+        # Need to use expand because of in-place inversion
+        # revert to decomposition once in-place inversion removed
+        base_decomp = op.base.expand().circuit
+    except qml.operation.DecompositionUndefinedError:
+        return None
+
+    return [Controlled(x, op.control_wires, work_wires=op.work_wires) for x in base_decomp]
```

Comment:
```suggestion
    return [Controlled(decomp_op, op.control_wires, work_wires=op.work_wires) for decomp_op in base_decomp]
```

Reply:
A more descriptive name

Reply:
`base_decomp` pushes it over the 100-character line length. Does `newop` work? That keeps it at 100.